RS policy references (#2447)
RedisLabs/redislabs-docs: content/rs/references/policies/redis-upgrade-policy.md

---
Title: Redis upgrade policy
linkTitle: redis_upgrade_policy
description: Redis Enterprise upgrade policy. 
weight: $weight
alwaysopen: false
toc: "true"
categories: ["RS"]
aliases: 
---

In version 6.2.4, Redis Enterprise Software introduced the Redis database compatibility upgrade policy (`redis_upgrade_policy`). This policy limits the [default Redis version]({{<relref "/rs/references/policies/default-redis-version">}}) when creating or updating databases.

Changes to the upgrade policy do not affect existing databases. The policy is used only when you create a new database, upgrade a database, or change its configuration.

> For best results, we recommend changing the policy value only after upgrading to a major release of Redis Enterprise Software.

Reviewer: ```suggestion
For best results, change the policy value only after upgrading to a major release of Redis Enterprise Software.
```

## Policy values

| Value | Description |
|-------|-------------|
| latest | Limits compatibility to the latest (most recent) version of open source Redis supported by your copy of Redis Enterprise Software (default behavior for Redis Enterprise versions earlier than v6.2.4) |
| major  | Limits Redis compatibility to major releases (default policy for Redis Enterprise v6.2.4 and later) |

As of v6.2.4, `redis_upgrade_policy` defaults to `major`, which limits Redis database compatibility to the most recent major release.

> If you change the policy to `latest`, you need to upgrade Redis Enterprise Software every time there’s a minor release. Further, you’ll need to leave the policy set to `latest` until the next major release of Redis Enterprise Software, which generally happens every 18-24 months.

Reviewer: ```suggestion
If you change the policy to `latest`, you need to upgrade Redis Enterprise Software with each minor release. Furthermore, leave the policy set to `latest` until the next major release of Redis Enterprise Software, which generally happens every 18-24 months.
```
